weave/trace_server/costs/cost_checkpoint.json

```diff
@@ -0,0 +1,4386 @@
+{
```

Comment:
Curious how your query logic works. lets say i have a call from time T, and a cost created at time T+1. If there are no other costs registered for that model/provider pair, do i use the "future" cost?

If the answer is "no", we might want to backfill this a bit.